scrapenews/spiders/businessday.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+
+from .sitemap import SitemapSpider
+from scrapenews.items import ScrapenewsItem
+from datetime import datetime
+import pytz
+
+SAST = pytz.timezone('Africa/Johannesburg')
+
+
+class BusinessDaySpider(SitemapSpider):
+    name = 'businessday'
+    allowed_domains = ['www.businesslive.co.za']
+
+    sitemap_urls = ['https://www.businesslive.co.za/sitemap.xml']
+    sitemap_follow = ['politics', 'companies', 'people', 'national', 'news', 'special-reports']
+
+    publication_name = 'Business Day'
+
+    def parse(self, response):
+        canonical_url = response.xpath('//link[@rel="canonical"]/@href').extract_first()
+        if canonical_url:
+            url = canonical_url
+        else:
+            url = response.url
+
+        title = response.xpath('//h1[@class="article-title article-title-primary"]/span/text()').extract_first()
+        self.logger.info('%s %s', url, title)
+        article_body = response.xpath('//div[@class="article-content  article-style-None"]')
```

Comment:
It looks like `.article-content` contains too much - it also contains links to other articles at the bottom which will have a negative impact on search results. Looks like this works much the same as `timeslive` because it's also a Tiso Blackstar publication and there we used `response.css('div.article-widget-text')`

![scrapenews-bdlive-too-much](https://user-images.githubusercontent.com/235801/41194964-b1842584-6c24-11e8-80c7-a3b2f370260a.png)

Reply:
Whoops! Ok, will fix.